src/lib.rs

```diff
@@ -339,8 +338,15 @@ impl LightningNode {
             .restart(Self::get_foreground_periods());
 
         // let data_store_clone = Arc::clone(&data_store);
-        // rt.handle()
-        //     .block_on(async { migrate_funds(&strong_typed_seed, data_store_clone, &sdk).await })?;
+        // let auth_clone = Arc::clone(&auth);
+        // fund_migration::migrate_funds(
+        //     rt.handle(),
+        //     &strong_typed_seed,
+        //     data_store_clone,
+        //     &sdk,
+        //     auth_clone,
+        //     &environment.backend_url,
+        // )?;
```

Comment:
Maybe add a comment as for why we comment this out

Reply:
Added todo comment in [3e2bf28](https://github.com/getlipa/lipa-lightning-lib/pull/650/commits/3e2bf286a007f49c9f637531a48c6b803f330262)